src/index.html

```diff
@@ -17,6 +30,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index/html"
```

Comment:
The `href` attribute for the logo link seems incorrect. It should likely be `href="index.html"` instead of `href="index/html"`.